src/ls_server.rs

```diff
@@ -61,8 +62,8 @@ enum Method {
 
 #[derive(Debug)]
 enum Notification {
-    CancelRequest(usize),
-    Change(ChangeParams),
+    CancelRequest(NumberOrString),
```

Comment:
I see that bruno already added the deserializaton for NumberOrString, I guess there's no need for me to make any changes.